### PR TITLE
✨ Add hideDownload & excluded wallet for bookstore list 

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: src/routes/likernft/book/store.ts [hunk 1]
```diff
@@ -43,12 +43,14 @@ router.get('/list', jwtOptionalAuth('read:nftbook'), async (req, res, next) => {
   try {
     const {
       wallet,
+      excluded_wallet: excludedWallet,
```
Comment (williamchong) [inline src/routes/likernft/book/store.ts]:
```suggestion
      exclude_wallet: excludedWallet,
```
Using verb seems better for qs

File: src/util/api/likernft/book/index.ts [hunk 2]
```diff
@@ -230,6 +231,7 @@ export async function listLatestNFTBookInfo({
 } = {}) {
   let snapshot = likeNFTBookCollection.orderBy('timestamp', 'desc');
   if (ownerWallet) snapshot = snapshot.where('ownerWallet', '==', ownerWallet);
+  if (excludedOwnerWallet) snapshot = snapshot.where('ownerWallet', '!=', excludedOwnerWallet);
```
Comment (williamchong) [inline src/util/api/likernft/book/index.ts:235]: Might need to build index

File: src/routes/likernft/book/store.ts (hunk 1 above)
Comment (nwingt) [reply src/routes/likernft/book/store.ts]: But are `wallet`, `limit`, and `key` also a noun?

File: src/util/api/likernft/book/index.ts (hunk 2 above)
Comment (nwingt) [reply src/util/api/likernft/book/index.ts:235]: built

File: src/routes/likernft/book/store.ts (hunk 1 above)
Comment (williamchong) [reply src/routes/likernft/book/store.ts]: it's like `ignore_whitespace=1` vs `ignored_whitespace=1`

**Review by williamchong — APPROVED** [review]

**Pull request merged**